WordPress/src/main/java/org/wordpress/android/util/encryption/LogsEncryptionActions.kt

```diff
@@ -0,0 +1,52 @@
+package org.wordpress.android.util.encryption
+
+import com.android.volley.NetworkResponse
+import com.android.volley.Request
+import com.android.volley.Response
+import com.android.volley.Response.ErrorListener
+import com.android.volley.Response.Listener
+import com.android.volley.VolleyError
+import com.android.volley.toolbox.HttpHeaderParser
+import com.android.volley.toolbox.JsonRequest
+import org.json.JSONObject
+import org.wordpress.android.WordPress
+import org.wordpress.android.util.AppLog
+
+class LogsEncryptionActions {
+    /**
+     * Sends a POST request with the encrypted logs and useful fields
+     * for the server to decrypt them.
+     */
+    fun sendEncryptedLogsFile(encryptedLogsJson: JSONObject) {
+        val jsonRequest = JsonStringRequest(
+                Request.Method.POST,
+                ENCRYPTION_LOGS_SERVER_URL,
+                encryptedLogsJson,
+                Listener { AppLog.i(AppLog.T.API, "Encrypted logs response successfully received from Wordpress API") },
+                ErrorListener { AppLog.e(AppLog.T.API, "An error occurred when sending logs. " + it.message) }
+        )
+        WordPress.sRequestQueue.add(jsonRequest)
```

Comment:
This call tightly binds this class to the `WordPress` one. This could make it difficult to implement goal 4 (migrating this to a shared library later).